test/unit/report_importer_test.rb

```diff
@@ -10,7 +10,7 @@ class ReportImporterTest < ActiveSupport::TestCase
   end
 
   test 'it should import reports with no metrics' do
-    r = ReportImporter.import(read_json_fixture('report-empty.json'))
+    r = ConfigReportImporter.import(read_json_fixture('report-empty.json'))
     assert r
     assert_equal({}, r.metrics)
```

Comment:
I thought the ConfigReportImporter was populating the metrics column.  What is populating the metrics column to be `{}`, prior to you adding the `rescue {}` into the metrics getter?